k6/src/k6/utils.ts

```
@@ -0,0 +1,102 @@
+                            return JSON.parse(responseBody).message.startTime;
+                        } catch (error) {
+                            console.error("Error parsing JSON message payload:" + message);
+                            throw new Error("Invalid JSON format");
+                        }
+                    } else { // Is a partial message
+                        return "HTTP/1.1 200" + message;
+                    }
+                });
+            // Check if last element is a number or a partial message
+            const lastMessage = messageParts[messageParts.length-1];
+            if (isNaN(lastMessage)) { // Is partial message
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)